### PR TITLE
Remove gfortran

### PR DESCRIPTION
**Pull request by ocefpaf**
Port the changes from `climlab` here to fix Fortran packages on Windows.

### CODE REVIEW

File: recipe/run_conda_forge_build_setup_win.bat [hunk 1]
```diff
@@ -25,6 +25,11 @@ if "%CI%" == "" (
     echo - %CI% >> .ci_support\%CONFIG%.yaml
 )
 
+:: Remove chocolatey gfortran.
+if "%CI%" == "azure" (
+  rm "C:\ProgramData\Chocolatey\bin\gfortran.exe"
```
Comment (isuruf) [inline recipe/run_conda_forge_build_setup_win.bat]: Can we remove `C:\ProgramData\Chocolatey\bin\` from path?

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

File: recipe/run_conda_forge_build_setup_win.bat (hunk 1 above)
Comment (ocefpaf) [reply recipe/run_conda_forge_build_setup_win.bat]: Probably. I'm not sure how to manipulate paths on Windows though.

File: recipe/run_conda_forge_build_setup_win.bat [hunk 2]
```diff
@@ -25,10 +25,20 @@ if "%CI%" == "" (
...
+set "PATH=%PATH:C:\Program Files\Git\usr\bin;=%"
+set "PATH=%PATH:C:\Program Files\Git\cmd;=%"
+set "PATH=%PATH:C:\Program Files\Git\mingw64\bin;=%"
+set "PATH=%PATH:C:\Program Files (x86)\Subversion\bin;=%"
+set "PATH=%PATH:C:\Program Files\CMake\bin;=%"
+set "PATH=%PATH:C:\Program Files\OpenSSL\bin;=%"
+set "PATH=%PATH:C:\Strawberry\c\bin;=%"
+set "PATH=%PATH:C:\Strawberry\perl\bin;=%"
+set "PATH=%PATH:C:\Strawberry\perl\site\bin;=%"
+set "PATH=%PATH:c:\tools\php;=%"
```
Comment (ocefpaf) [inline recipe/run_conda_forge_build_setup_win.bat]: AppVeyor was cleaner on this regard. Lots of pre-installed things, little interference.

Comment (isuruf) [reply recipe/run_conda_forge_build_setup_win.bat]: Appveyor had lots of things pre-installed, but they didn't add those to PATH.

Comment (ocefpaf) [reply recipe/run_conda_forge_build_setup_win.bat]: Yeah. That is what I meant with my rant above. AppVeyor have tons of experience on Windows stuff that Azure could use ;-p

**Pull request merged**